lib/godwoken_explorer/graphql/resolovers/transaction.ex

```diff
@@ -230,6 +231,28 @@ defmodule GodwokenExplorer.Graphql.Resolvers.Transaction do
     end
   end
 
+  defp query_with_method_id_name({:error, _} = error, _input), do: error
+
+  defp query_with_method_id_name(query, input) do
+    method_id = Map.get(input, :method_id)
+    method_name = Map.get(input, :method_name)
+
+    query =
```

Comment:
`method_id` and `method_name` should be mutually exclusive, and `method_id` should be prior to `method_name` cuz every `method_name` has its `method_id` but not every `method_id` has its `method_name`

Reply:
fixed